src/convertToObject.js

```diff
@@ -12,7 +12,13 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').map(el => el.slice(3).split(': '))
+    .reduce((sum, curent) => {
```

Comment:
sum is a bad name. we are not really summing anything.

res or obj would be better.